app/templates/index.hbs

```diff
@@ -1,12 +1,21 @@
 <div class="{{styles.indexRoute}} {{vudu 'liquidInner'}}" id="index-route-scroll-context">
   {{#index-scroll-container indexSubSection=s}}
-    {{#each model as |section|}}
-      {{#way-point
-        action=(action "didHitWaypoint" section.waypoint null)
-        contextElementId="index-route-scroll-context"
-        class=section.className }}
-        {{{section.copy}}}
-      {{/way-point}}
+    {{#each model as |group|}}
+      {{#scroll-section
+        class=(concat 'index--' group.title)
+        action=(action "didHitWaypoint" group.title null)
+      }}
+        <p class={{vudu 'pb5'}}>
+          {{#each group.combined as |c|}}
+            <span class={{if c.break "relative flex" "relative"}}>
```

Comment:
the way around this w/o classic CSS is:

```
<span class={{if c.break (concat v.relative " " v.flex) v.relative}}>
```

or you can declare classes in the `route/index.js` like:

```
let local = v({
  scrollSectionBreak: {
    "@composes: [v.relative, v.flex]
  }
})
```


<span class={{if c.break local.scrollSectionBreak v.relative}}>
```